### DISCUSSION TITLE
Adding Power Port Type C21/C22

### DISCUSSION THREAD

**Discussion by scotsie** [netbox-community/netbox#6876] [Q&A] [answered]
Working with new Dell MX7000 chassis and it uses C22 power ports in it's PSUs. Perhaps my searches have failed in the documentation but is there a method to add more power ports time or is that relegated to feature requests?
https://en.wikipedia.org/wiki/IEC_60320

If someone can point me in a direction or provide a link it would be appreciated. I did find where I can create a yaml template for the chassis and would be happy to give that a shot. Fairly new to the product. (handed a login yesterday and tasked with converting several datacenters from several spreadsheets).

Thanks in advance,

**Comment by jeremystretch** [comment, marked as answer]
Power port and outlets are defined statically, but we can certainly add these types. Just need a [feature request](https://github.com/netbox-community/netbox/issues/new?assignees=&labels=type%3A+feature&template=feature_request.yaml) citing your use case; copying & pasting what you have above should be fine.

**Reply by scotsie** [reply]
I appreciate the guidance and quick response. I have opened feature request #6883 and cleaned up my many typos.
